src/prop/minisat/core/Solver.h

```diff
@@ -176,6 +165,17 @@ class Solver : protected EnvObj
   */
  bool needProof() const;
 
+ /*
+  * Returns true if the solver should add all clauses at the current assertion
+  * level.
+  *
+  * FIXME: This is a workaround. Currently, our resolution proofs do not
```

Comment:
Open a projects issue for this? Or is this fixed now?

Reply:
It is fixed only for proofs. For assumptions-based unsat cores for example it's still used. I'm not sure if it could be removed for that as well. I'll open a project issues to try to determine this.